### PR TITLE
inital k6 scripts

### PR DESCRIPTION
**Pull request by tedhtchang**
Signed-off-by: ted chang <htchang@us.ibm.com>
Initial k6 scripts examples for running performance test
Initial Grafana Dashboard export example

### CODE REVIEW

File: k6_test/README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# k6 ModelMesh Scripts Quick Start
+## Perquisites
```
Comment (aluu317) [inline k6_test/README.md]:
```suggestion
## Prerequisites
```

File: k6_test/README.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+1. Install [Modelemsh-serving](https://github.com/kserve/modelmesh-serving/blob/main/docs/quickstart.md)
```
Comment (aluu317) [inline k6_test/README.md]:
```suggestion
1. Install [Modelmesh-serving](https://github.com/kserve/modelmesh-serving/blob/main/docs/quickstart.md)
```

**Comment by aluu317** [comment]
There's an empty file [docs/prometheus/README.md](https://github.com/kserve/modelmesh-performance/pull/1/files#diff-a0b539ddd054f827681cd29412de2f5eb07c0975a6e2b6665aa6ccc71249f86c). Did you intend to add something?

**Review by aluu317 — APPROVED** [review]
Initial scripts look good. We're going to refactor pretty soon so I didn't do a read through of everything line by line

**Comment by tedhtchang** [comment]
> Initial scripts look good. We're going to refactor pretty soon so I didn't do a read through of everything line by line

Thanks. Please go ahead and refactor.
The initial script were meant for quick demo.

**Pull request merged**